src/components/LeftSidebar/SearchConversationsResults/SearchConversationsResults.vue

```diff
@@ -0,0 +1,332 @@
+<!--
+  - SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<script setup lang="ts">
+import { computed, ref } from 'vue'
+import { RecycleScroller } from 'vue-virtual-scroller'
+
+import IconChatPlus from 'vue-material-design-icons/ChatPlus.vue'
+
+import { loadState } from '@nextcloud/initial-state'
+import { t } from '@nextcloud/l10n'
+
+import NcAppNavigationCaption from '@nextcloud/vue/components/NcAppNavigationCaption'
+import NcListItem from '@nextcloud/vue/components/NcListItem'
+
+import AvatarWrapper from '../../AvatarWrapper/AvatarWrapper.vue'
+import ConversationIcon from '../../ConversationIcon.vue'
+import Hint from '../../UIShared/Hint.vue'
+import Conversation from '../ConversationsList/Conversation.vue'
+import { useSettingsStore } from '../../../stores/settings.js'
+
+import { ATTENDEE, CONVERSATION, AVATAR } from '../../../constants.ts'
+import type { Conversation as TypeConversation, ParticipantSearchResult } from '../../../types/index.ts'
+
+const isCirclesEnabled = loadState('spreed', 'circles_enabled')
+const canStartConversations = loadState('spreed', 'start_conversations')
+const settingsStore = useSettingsStore()
+const isCompact = computed(() => settingsStore.conversationsListStyle === CONVERSATION.LIST_STYLE.COMPACT)
+
+const props = defineProps<{
+	searchText: string,
+	conversationsList: Array<TypeConversation>,
+	contactsLoading: boolean,
+	searchResultsListedConversations: Array<TypeConversation>,
+	searchResults: Array<ParticipantSearchResult>,
+}>()
+
+const emit = defineEmits<{
+	(event: 'abort-search'): void,
+	(event: 'create-new-conversation', searchText: string): void,
+	(event: 'create-and-join-conversation', item: TypeConversation): void,
+}>()
+
+/* Consider:
+* avatar size (and two lines of text) or compact mode (28px)
+* list-item padding
+* list-item__wrapper padding
+*/
+const itemSize = computed(() => isCompact.value ? 28 + 2 * 2 + 0 * 2 : AVATAR.SIZE.DEFAULT + 2 * 4 + 2 * 2)
```

Comment:
```suggestion
/* Consider:
 * avatar size (and two lines of text) or compact mode (28px)
 * list-item padding
 * list-item__wrapper padding
 */
const itemSize = computed(() => isCompact.value ? 28 + 2 * 2 + 0 * 2 : AVATAR.SIZE.DEFAULT + 2 * 4 + 2 * 2)
```

Reply:
Or to clarify what is what here

```suggestion
// Item's content (avatar) + internal_padding * 2 + external_padding * 2
const itemSize = computed(() => isCompact.value ? 28 + 2 * 2 + 0 * 2 : AVATAR.SIZE.DEFAULT + 2 * 4 + 2 * 2)
```